config.json

```diff
@@ -509,7 +509,12 @@
       "core": false,
       "difficulty": 6,
       "slug": "minesweeper",
-      "topics": null,
+      "topics": [
+        "integers",
+        "matrices",
+        "games",
+        "conditionals"
```

Comment:
Great suggestions! :) Maybe also `strings` and `lists`? :)

Reply:
OKK